src/platform/lumin-runtime/platform-factory.js

```diff
@@ -475,6 +550,15 @@ export class PlatformFactory extends NativeFactory {
     if (this.isController(child)) {
```

Comment:
Do we need child prism controllers anymore? How are those used in practice? I thought it was by adding a Scene element as a child but doesn't Scene mean something else now (the top-level holder of multiple Prisms)?

This is unchanged code so not something that has to be addressed in this review -- I'm just trying to understand if we can get rid of it all and simplify a bit. If there is another way to get the onEvent/onUpdate callbacks, I really think we should simplify and drop child controllers.

Reply:
No, we don't need the prism controllers support anymore. I would create a new issue for refactoring - removing unused code related to PrismContoller.